### PR TITLE
OHRM5X-2469: Bump OrangeHRM, MariaDB versions

### PR DESCRIPTION
**Pull request by devishke-orange**
Changes:
- Updated OrangeHRM version to 5.6
- Updated MariaDB version to 10.11.6
- Removed GPLv2 license text and warnings
- Removed bc dependency
- Bump package version to 0.0.4

### CODE REVIEW

**Comment by devishke-orange** [comment]
@RajithaKumara please review and merge

File: aws/compose.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ services:
     container_name: orangehrm
 
   mariadb1011:
-    image: mariadb:10.11.4
+    image: mariadb:10.11.6
```
Comment (RajithaKumara) [inline aws/compose.yml:18]: do we need the patch version?

File: aws/orangehrm [hunk 2]
```diff
@@ -54,10 +54,7 @@ export ORANGEHRM_WEBROOT_VOLUME="orangehrm_orangehrm-webroot"
 export ORANGEHRM_DB_VOLUME="orangehrm_orangehrm-db"
 
 # Update URL
-export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/os-cloud/main/aws/compose.yml"
-
-# License changed version
-export NEW_LICENSE_VERSION="5.6"
+export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/orangehrm-os-cloud-packages/main/aws/compose.yml"
```
Comment (RajithaKumara) [inline aws/orangehrm:57]: hope all places changed related to repo name change

File: aws/scripts/install [hunk 3]
```diff
@@ -191,17 +187,17 @@ update_configuration() {
 
 	installer_command_file="/var/www/html/installer/Command/InstallOnNewDatabaseCommand.php"
 
-  for line in {343..357}; do
-    docker exec orangehrm sed -i "343 d" $installer_command_file
+  for line in {342..356}; do
```
Comment (RajithaKumara) [inline aws/scripts/install:190]: will discuss this offline

**Review by RajithaKumara — APPROVED** [review]

File: aws/compose.yml (hunk 1 above)
Comment (devishke-orange) [reply aws/compose.yml:18]: I decided to keep in the event that another subversion breaks the installation

File: aws/orangehrm (hunk 2 above)
Comment (devishke-orange) [reply aws/orangehrm:57]: changed

File: aws/scripts/install (hunk 3 above)
Comment (devishke-orange) [reply aws/scripts/install:190]: this needs to be changed. we can add arguments to installer like the upgrader. or we can take from env variables as Smanatha suggested https://github.com/orangehrm/orangehrm/issues/1776

**Pull request merged**